include/ode_solvers/oracle_functors.hpp

```diff
@@ -392,4 +398,105 @@ struct HessianFunctor {
 
 };
 
+
+struct AutoDiffFunctor
```

Comment:
Coding style should be fixed here

Reply:
Are you talking about the code format or software best practice to write elegant and readable code?

Reply:
both, but primarily the former.